solr/core/src/java/org/apache/solr/cloud/ZkCLI.java

```diff
@@ -424,30 +419,27 @@ public static void main(String[] args)
             System.exit(1);
           }
 
-          String path = arglist.get(0).toString();
-          InputStream is = new FileInputStream(arglist.get(1).toString());
-          byte[] data = IOUtils.toByteArray(is);
+          String path = arglist.get(0);
+          byte[] data;
+          try (InputStream inputStream = Files.newInputStream(Path.of(arglist.get(1)))) {
+            data = inputStream.readAllBytes();
+          }
```

Comment:
Instead use `Files.readAllBytes(path)`

Reply:
Addressed in 7079308294cb7f4d4c468dbdc890afd2a6d67d2e